pkgs/applications/blockchains/lightwalletd/default.nix

```diff
@@ -0,0 +1,37 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "lightwalletd";
+  version = "0.4.7";
+
+  src = fetchFromGitHub {
+    owner = "zcash";
+    repo  = "lightwalletd";
+    rev = "v${version}";
+    sha256 = "0dwam3fhc4caga7kjg6cc06sz47g4ii7n3sa4j2ac4aiy21hsbjk";
+  };
+
+  vendorSha256 = null;
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/zcash/lightwalletd/common.Version=v${version}"
+    "-X github.com/zcash/lightwalletd/common.GitCommit=a4cc08b89eddd88befc36931422c53e4d4190185"
```

Comment:
That won't get updated. Can we just set this to the version or blank instead?

Reply:
That's why I originally set `rev` to the commit hash, as that would enable auto-updating. I can set `GitCommit` to the version string for now and deal with it later if someone complains.